### PR TITLE
Added instructions to deploy

### PR DESCRIPTION
**Pull request by tasomaniac**
I'm barely a pro at this but I had a little bit of knowledge. 
Here are the steps that I did to apply the changes and deploy the website. 

Also, do not ignore `package.json` file to make easier for people to install dependencies.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "gulp-util": "^3.0.8",
+    "gulp-if": "^2.0.2",
+    "gulp-watch": "^4.3.11"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/novoda/novoda.github.io.git"
```
Comment (juankysoriano) [inline package.json:22]: @tasomaniac I am not an expert on this thing neither, I have never seen this notation before. "https://blabla.git". I am used to simply see `https://` or `git://` within the `repository:url` of a `package.json` for `npm`. 

Is this notation correct? I am curious.

Comment (tasomaniac) [reply package.json:22]: I dunno :) It was auto-generated when I do `npm init` or something like that. 
I think it is not super important but I can also change to whatever you want.

Comment (juankysoriano) [reply package.json:22]: @tasomaniac don't worry about changing it, if it is autogenerated, I was just curious.

**Review by juankysoriano — APPROVED** [review]
LGTM 👍  thanks for the instructions!

**Comment by tasomaniac** [comment]
Wohoo. Let's now hope that the instructions will make easier to deploy in the future.

**Pull request merged**